### ISSUE TITLE
How long does it take to complete training?

### ISSUE THREAD

**Issue by weinixuehao** [xavysp/DexiNed#85]
It took two days to train with my own dataset, see below for loss and epoch.
![image](https://user-images.githubusercontent.com/17869361/125713745-a53e96cb-11b7-437e-8861-f74c9f482a69.png)
I use the default epoch is 24 so it still needs much time to complete the training?
How much loss is yours when the training is done?

**Comment by weinixuehao** [comment]
My computer is an iMac so there is no GPU card, I am afraid it takes so much time to complete the training.

**Comment by xavysp** [comment]
Hi, using a GPU  almost 2 days. But if you are doing similar work, try fine-tuning, from DexiNed-BiPED checkpoint

**Comment by weinixuehao** [comment]
1、How to fine-tuning? 
2、Make fine-tuning directly based on the new training data and the DexiNed-BiPED checkpoint without change any code?
3、Do you mean that the result of fine-tuning is better than training from scratch?

**Comment by weinixuehao** [comment]
![results](https://user-images.githubusercontent.com/17869361/126090583-91207d0a-98e5-4209-b02f-83be88ccbb53.png)
I feel that the effect of the fourth picture is better, still need to continue training?

**Comment by xavysp** [comment]
> ![results](https://user-images.githubusercontent.com/17869361/126090583-91207d0a-98e5-4209-b02f-83be88ccbb53.png)
> I feel that the effect of the fourth picture is better, still need to continue training?
> What is wrong with the line breaking？

Sorry you are training in DexiNed-Pytorch aren't you?
Did you updated to the las DexiNed Version

**Comment by weinixuehao** [comment]
@xavysp I am training in TF-2

**Comment by xavysp** [comment]
I strongly recommend you to use DexiNed-Pytorch. Since the TF-2 version need some improvements.
